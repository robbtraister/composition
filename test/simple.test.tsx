'use strict'

/* global expect, test */

import React from 'react'
import { render } from '@testing-library/react'

import MyComponent from '../src'

test('simple test', async () => {
  const { asFragment } = render(<MyComponent />)
  expect(asFragment()).toMatchSnapshot()
})
