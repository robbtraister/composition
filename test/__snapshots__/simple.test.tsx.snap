// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`simple test 1`] = `
<DocumentFragment>
  <div>
    test
  </div>
</DocumentFragment>
`;
